test/e2e/collect-ci-artifacts.sh

```diff
@@ -0,0 +1,27 @@
+#! /bin/bash
+
+set -o pipefail
+set -o nounset
+set -o errexit
+
+: "${KUBECONFIG:?}"
+: "${ARTIFACT_DIR:?}"
+: "${KUBECTL:=oc}"
```

Comment:
I'm running into issues where "oc" exists in the CI container image, but for whatever reason it can't be found from the $PATH. When diving into this further, it looks like there's a /cli/oc binary present, which explicitly overriding the KUBECTL variable in our o/release configuration could be a decent middle ground here.